src/ctia/stores/es/mapping.clj

```diff
@@ -122,6 +122,8 @@
 (def related-actors
   {:properties (assoc related
                       :actor_id token)})
+(def related-assets
```

Comment:
not sure this is needed anymore, this pattern exists before we moved on to STIX 2.0 influenced schemas and the Relationship model.

Reply:
Good to know. Honestly, I still don't exactly know what I'm doing. I just copy-pasta a bunch of things and changed it slightly.